app/controllers/api_docs/services_controller.rb

```diff
@@ -42,6 +42,7 @@ def initialize(name, system_name)
     def json
       parsed_content = JSON.parse(file_content)
       parsed_content['servers'] = [{ 'url' => backend_base_host }] if backend_api?
+      parsed_content['components'] = parsed_content.delete('components') if parsed_content.key?('components')
```

Comment:
I was able to reproduce the error, but this doesn't fix it for me. I see the same result with or without this change. Only manually modifying the JSON to actually placing `components` after `paths` works for me.

Anyway, IMO setting `components` as the last key in the ruby hash isn't the right approach for the issue, because even if it worked, it wouldn't fix the problem, only hide the consequences. The right solution is to find out why JSON attributes order matters and fix the code so attributes order doesn't matter.